### ISSUE TITLE
Batch_norm and Dropout kept on while testing

### ISSUE THREAD

**Issue by m273033** [kwotsin/TensorFlow-ENet#33]
I have tried with keeping them on and off during the testing by setting the flag "is_training" in test_enet.py. However, I get better performance while keeping them off and probably keeping them on gives false results.
